bemani/backend/jubeat/clan.py

```diff
@@ -1164,6 +1160,8 @@ def format_scores(self, userid: UserID, profile: Profile, scores: List[Score]) -
                 music.replace_dict(str(score.id), data)
 
         for scoreid in music:
+            if int(scoreid) >= max_music_id or int(scoreid) <= min_music_id:
```

Comment:
Having equals in both of these seems like a problem. Wouldn't that disregard the exact scoreid when comparing?